Include linux/arm64 build

adduc opened the pull request:
Hi,

I'm not as familiar with golang best practices for Makefiles, and would open to making additional changes if needed.

For context, I'm looking to bring up an instance of prometheus-msteams on one of Amazon's ARM instances. While I am able to compile the binary locally, being able to download each release direct from GitHub would streamline the process.

I was able to build linux/arm64 binaries with golang 1.13.7 (the version used for Travis), and was able to build windows/arm64 and darwin/arm64 with more recent versions of golang.

Knappek commented:
closes #152

Knappek commented:
@adduc thanks for this PR! I would say we should upgrade our Go version to `1.16`. What do you think, should we do this in this PR as well (as it might make the Makefile easier) or in a separate PR?

Knappek commented:
@adduc we should also create a linux-arm based Docker image. Maybe we can leverage [docker buildx](https://www.docker.com/blog/multi-arch-build-and-images-the-simple-way/) in order to achieve that?

adduc commented:
> @adduc thanks for this PR! I would say we should upgrade our Go version to `1.16`. What do you think, should we do this in this PR as well or in a separate PR?

I've updated the Go version to 1.16.4, and added Darwin ARM64 builds to the Makefile and Travis-managed releases.

> we should also create a linux-arm based Docker image. Maybe we can leverage docker buildx in order to achieve that?

I am open to tackle this in the next day or two.

choeffer commented:
Any news on this? See #350 . I have stumbled across this also, and was really confused, that this is still an issue today.
